tests/vmi_multiqueue_test.go

```diff
@@ -123,5 +123,35 @@ var _ = Describe("[Serial]MultiQueue", func() {
 			}
 		})
 
+		It("should be able to create a multi-queue VMI when requesting a single vCPU", func() {
+			vmi := tests.NewRandomFedoraVMIWitGuestAgent()
```

Comment:
Creating a fedora VM in tests is the last resort, if nothing else works. Here you can savely use cirros.

Reply:
Right; I had just copied from the previous example.

Cirros is good enough here; thanks for spotting.